eng/packaging.targets

```diff
@@ -32,6 +32,7 @@
   <PropertyGroup Condition="'$(ServicingVersion)' != ''">
     <!-- Always update the package version in servicing. -->
     <PackageVersion>$(MajorVersion).$(MinorVersion).$(ServicingVersion)</PackageVersion>
+    <PackageVersion Condition="'$(VersionSuffix)' != ''">$(PackageVersion)-$(VersionSuffix)</PackageVersion>
```

Comment:
Why not instead use `VersionPrefix` property rather than trying to mimic and duplicate what arcade does?  https://github.com/dotnet/arcade/blob/main/src/Microsoft.DotNet.Arcade.Sdk/tools/Version.BeforeCommonTargets.targets#L163